packages/hdwallet-provider/src/index.ts

```diff
@@ -7,15 +7,19 @@ import { Transaction, FeeMarketEIP1559Transaction } from "@ethereumjs/tx";
 import Common from "@ethereumjs/common";
 
 // @ts-ignore
-import ProviderEngine from "@trufflesuite/web3-provider-engine";
-import FiltersSubprovider from "@trufflesuite/web3-provider-engine/subproviders/filters";
-import NonceSubProvider from "@trufflesuite/web3-provider-engine/subproviders/nonce-tracker";
-import HookedSubprovider from "@trufflesuite/web3-provider-engine/subproviders/hooked-wallet";
-import ProviderSubprovider from "@trufflesuite/web3-provider-engine/subproviders/provider";
+import ProviderEngine from "web3-provider-engine";
 // @ts-ignore
-import RpcProvider from "@trufflesuite/web3-provider-engine/subproviders/rpc";
+import FiltersSubprovider from "web3-provider-engine/subproviders/filters";
 // @ts-ignore
-import WebsocketProvider from "@trufflesuite/web3-provider-engine/subproviders/websocket";
+import NonceSubProvider from "web3-provider-engine/subproviders/nonce-tracker";
+// @ts-ignore
+import HookedSubprovider from "web3-provider-engine/subproviders/hooked-wallet";
+// @ts-ignore
+import ProviderSubprovider from "web3-provider-engine/subproviders/provider";
+// @ts-ignore
+import RpcProvider from "web3-provider-engine/subproviders/rpc";
+// @ts-ignore
+import WebsocketProvider from "web3-provider-engine/subproviders/websocket";
```

Comment:
The current subprovider declaration files ([filters.d.ts and co](https://github.com/trufflesuite/truffle/blob/bump-web3-provider-engine/packages/hdwallet-provider/typings/web3-provider-engine/subproviders/filters.d.ts#L1)) should be updated to drop the  **trufflesuite** scope. This should allow removing the _ts-ignore_ directive